src/main/kotlin/com/workos/users/UsersApi.kt

```diff
@@ -735,4 +733,61 @@ class UsersApi(private val workos: WorkOS) {
     val config = RequestConfig.builder().data(options).build()
     return workos.post("/users/${options.userId}/send_verification_email", MagicAuthChallenge::class.java, config)
   }
+
+  /**
+   * Parameters for the [updateUserPassword] method.
+   */
+  @JsonInclude(JsonInclude.Include.NON_NULL)
+  class UpdateUserPasswordOptions @JvmOverloads constructor(
+    @JsonProperty("userId") val userId: String,
+    @JsonProperty("password") val password: String
+  ) {
+    init {
+      require(userId.isNotBlank()) { "User id is required" }
+      require(password.isNotBlank()) { "Password is required" }
+    }
+
+    /**
+     * Builder class for [UpdateUserPasswordOptions].
+     */
+    class UpdateUserPasswordOptionsBuilder {
+      private lateinit var userId: String
+      private lateinit var password: String
+
+      /**
+       * Sets the user id.
+       */
+      fun userId(value: String) = apply { this.userId = value }
+
+      /**
+       * Sets the password.
+       */
+      fun password(value: String) = apply { this.password = value }
+
+      /**
+       * Creates an [UpdateUserPasswordOptions] with the given builder parameters.
+       */
+      fun build(): UpdateUserPasswordOptions {
+        return UpdateUserPasswordOptions(userId, password)
+      }
+    }
+
+    /**
+     * @suppress
+     */
+    companion object {
+      @JvmStatic
+      fun builder(): UpdateUserPasswordOptionsBuilder {
+        return UpdateUserPasswordOptionsBuilder()
+      }
+    }
+  }
+
+  /**
+   * Updates the password of a specified user.
+   */
+  fun updateUserPassword(updateUserPasswordOptions: UpdateUserPasswordOptions): User {
+    val config = RequestConfig.builder().data(updateUserPasswordOptions).build()
+    return workos.post("/users/${ updateUserPasswordOptions.userId}/password", User::class.java, config)
```

Comment:
I think this one should be a PUT.
```suggestion
    return workos.put("/users/${updateUserPasswordOptions.userId}/password", User::class.java, config)
```

Reply:
you're right, I even made a mental note of that and somehow still didn't make it put